pages/Using Rollbax in Plug-based applications.md

```diff
@@ -34,7 +34,7 @@ defp handle_errors(conn, %{kind: kind, reason: reason, stack: stacktrace}) do
     "request" => %{
       "cookies" => conn.req_cookies,
       "url" => "#{conn.scheme}://#{conn.host}:#{conn.port}#{conn.request_path}",
-      "user_ip" => (conn.remote_ip |> Tuple.to_list() |> Enum.join(".")),
+      "user_ip" => :inet_parse.ntoa(conn.remote_ip),
```

Comment:
This returns chardata which would break JSON, wouldn't it? Should be better to do `conn.remote_ip |> :inet.ntoa() |> List.to_string()` I think. Also, why `:inet_parse` and not `:inet`?